bin/wait-for-build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Had to still copy this one over as there was no way to call it directly in the package. Mostly because of line `11`, it uses a relative path in the original, which wouldn't work for us.